elvis.config

```diff
@@ -3,14 +3,28 @@
   elvis,
   [
    {config,
-    [#{dirs => ["apps/*/src", "apps/*/test", "test/ejabberd_tests/tests"],
+    [#{dirs => ["apps/*/src"],
        filter => "*.erl",
        ruleset => erl_files,
        rules => [{elvis_style, line_length, #{limit => 100,
                                               skip_comments => false}},
 		 {elvis_style, dont_repeat_yourself, #{min_complexity => 20}}
 		]
-      }
+      },
+     #{dirs => ["apps/*/test", "test/ejabberd_tests/tests"],
+       filter => "*.erl",
+       ruleset => erl_files,
+       rules => [{elvis_style, line_length, #{limit => 100,
+                                              skip_comments => false}},
+                 {elvis_style, dont_repeat_yourself, disable},
```

Comment:
Same as god_modules - either elvis will end up not being happy or I'll have to refactor several cases in mam_SUITE. Also, this doesn't seem right at some degree. Test cases are some stories. Those stories may be similar, but the moment I start to make generic helpers, those stories won't be as easy to read as they are now.